library/blivet.py

```diff
@@ -120,6 +120,7 @@
 use_partitions = None  # create partitions on pool backing device disks?
 disklabel_type = None  # user-specified disklabel type
 safe_mode = None       # do not remove any existing devices or formatting
+packages_only = None   # only set things up enough to get a list of required packages
```

Comment:
This commit does not seem to fit in this PR and it seems that the added functionality is not used or tested. Not sure what it is doing here.

Reply:
Sure, I can move this out.